plan/atlantis.go

```diff
@@ -0,0 +1,131 @@
+package plan
+
+import (
+	"fmt"
+	"sort"
+)
+
+type Atlantis struct {
+	Enabled  bool
+	Projects []AtlantisProject
+
+	// Profiles is a map of profile name -> role infro
+	Profiles map[string]AWSRole
```

Comment:
AWSProfiles maybe? I assume we might have to do similar things for other types of providers, should this be
```
Providers {
AWS: {}
}
```
to mirror how the v2 config specifies providers?